src/views/product-tutorials-view/components/product-view-content/components/branded-callout/branded-callout.module.css

```diff
@@ -35,7 +35,8 @@
 .productPattern {
   display: none;
 
-  @media (--dev-dot-tablet-up) {
+  /** Using a custom mq here: The pattern looked 'broken' with the sidebar / sidecar layout at sizes <1200px */
+  @media only screen and (min-width: 1280px) {
```

Comment:
In general, I find the `--dev-dot-desktop` breakpoint to be too low (1001px?) and imo it looks really off to still have the sidecar render at this size. 

If we didn't render the sidecar at ~1200... then we wouldn't need to use this override. Open to discussion here.

Reply:
<img width="1023" alt="Screen Shot 2022-05-16 at 3 36 26 PM" src="https://user-images.githubusercontent.com/36613477/168693566-604c71e4-b275-4c4e-942a-e0547ec0ec1c.png">
 picture from prod dev portal for reference

Reply:
This might solve this for you! https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/Dev-Portal?node-id=9688%3A139397. Also tagged you on a video on [Slack](https://hashicorp.slack.com/archives/C01KCU4HDPY/p1652821970693979?thread_ts=1652796652.219419&cid=C01KCU4HDPY)